### ISSUE TITLE
do you have 13.3.1(17d50)?

### ISSUE THREAD

**Issue by liduo1997** [iGhibli/iOS-DeviceSupport#92]

**Comment by mehulparmar4ever** [comment]
NO, I think they don't have added it yet. I also need **13.3.1(17d50)**

**Comment by Chaffy-0** [comment]
Sweet, I'm not the only one.

**Comment by bakhadev** [comment]
Anyone resolve this problem?

**Comment by mehulparmar4ever** [comment]
Hello everyone,

**Best solution, I have install macOS in my system & Now I am using Xcode 11.3.1**

The process of installing macOS 10.15 Catalina on an unsupported Mac explain in below YouTube video. If you need any help, feel free to contact me.

![Screenshot 2020-03-09 at 11 25 24 AM](https://user-images.githubusercontent.com/7687920/76186568-b08c7480-61f8-11ea-87de-e579051b4770.png)

Reg link: http://dosdude1.com/catalina/
https://www.youtube.com/watch?v=wxFzZWqoCng

Happy Coding!! Best of luck everyone.

With best,
Mehul
Sr. iOS Developer (Open for Remote Opportunities - Full-time/Contract)
https://about.me/mehul_parmar
Skype: mehulk.parmar

**Comment by harking** [comment]
Just copy 13.3 to 13.3.1 in your DeviceSupport directory.
`cp -r 13.3\ \(17C54\) 13.3.1\ \(17d50\)`

**Comment by Chaffy-0** [comment]
> Just copy 13.3 to 13.3.1 in your DeviceSupport directory.
> `cp -r 13.3\ \(17C54\) 13.3.1\ \(17d50\)`

Tried that but it didn't fix it.  Does 17d50 need to be a sub directory of 13.3.1?

**Comment by XinqingHappy** [comment]
I also need 13.3.1(17d50)

**Comment by XinqingHappy** [comment]
Pro test success => Just copy 13.3 (17C54) to the DeviceSupport directory. Name changed to 13.3.1 (17d50)

**Comment by ryanhuh** [comment]
It work for me!
XCode Version 10.1 (10B61)
IOS 13.3.1 (17d50)

copy from 13.3 (17C54) to 13.3.1 (17d50) 
and reboot XCode!

thanks all!
